doc/api/deprecations.md

```diff
@@ -2214,6 +2214,22 @@ the `_handle` property of the `Cipher`, `Decipher`, `DiffieHellman`,
 Using the `_handle` property to access the native object is deprecated because
 improper use of the native object can lead to crashing the application.
 
+<a id="DEP0118"></a>
```

Comment:
1. We usually do not assign a number until the PR is landed, we just add a temporal placeholder like `DEP0XXX`. Because some other deprecation PR can be landed before yours and the number will become outdated.